createdisk.sh

```diff
@@ -287,16 +287,19 @@ ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo podman pull quay.io/crcont
 # Stop the kubelet service so it will not reprovision the pods
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo systemctl stop kubelet
 
+# Enable the io.podman.socket service
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo systemctl enable io.podman.socket
+
 # Remove all the pods from the VM
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo crictl stopp $(sudo crictl pods -q)'
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo crictl rmp $(sudo crictl pods -q)'
 
 # Remove pull secret from the VM
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo rm -f /var/lib/kubelet/config.json'
 
-# Download the hyperV daemons dependency on host
+# Download the hyperV daemons and libvarlink-util dependency on host
 mkdir $1/hyperv
-sudo yum install -y --downloadonly --downloaddir $1/hyperv hyperv-daemons
+sudo yum install -y --downloadonly --downloaddir $1/hyperv hyperv-daemons libvarlink-util
```

Comment:
You add the package to download, but you aren't installing it into the image with:

```
${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo rpm-ostree install *.rpm'
```
 ?

Reply:
@gbraad it is downloaded in same directory so all the `*.rpm` should be installed.

Reply:
I notice, but in that case please rename paths:

https://github.com/code-ready/snc/blob/846938326d2184545866d823738365c72d674c16/createdisk.sh#L297-L308

This shouldbe called `packages` or similar and not `hyperv` as this gives the wrong idea for future maintenance

Reply:
@gbraad done.